### PR TITLE
Added recognition program

### LINKED ISSUE #4711: [Handbook] Add badges.layer5.io to Repository Overview
### Current Behavior
The badges.layer5.io site is nearly complete. It's time to list it in the Community Handbook.

### Desired Situation
Ideally, an entry is listed here - https://layer5.io/community/handbook/repository-overview

Perhaps, there is a Recognition Program logo that @ritiksaxena124 might share.

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by gopivaibhav**
**Description**

This PR fixes #4711

**Notes for Reviewers**
I have added information on the project in the Frontend projects. Feel free to let me know if I have to change anything here.

Note- 
I am not able to run the project locally even though I have installed the libraries. I guess some dependencies are missing in package.json file. Let me know if everything is working smoothly.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/assets/images/recognition-program/Recognition Program.svg
Comment (thisiskaransgit) [inline src/assets/images/recognition-program/Recognition Program.svg:1]: @gopivaibhav Prog ram?

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@gopivaibhav thanks, the build is failing since you forgot a hyphen in the image's filename, please fix it

File: src/assets/images/recognition-program/Recognition Program.svg
Comment (gopivaibhav) [reply src/assets/images/recognition-program/Recognition Program.svg:1]: This SVG was given by @ritiksaxena124 And I directly uploaded the image. So, I didn't give a look at the image file name. Also, I renamed the file now. Thanks !!

**Comment by l5io** [comment]
🚀 Preview for commit 69ce6dd2aed3f56e43532d4e0db6c5c9560d1bcc at: https://64d3898e1932315d9bf107c4--layer5.netlify.app

File: src/assets/images/recognition-program/Recognition Program.svg
Comment (gopivaibhav) [reply src/assets/images/recognition-program/Recognition Program.svg:1]: Added PNG instead of SVG for removing the space.

**Comment by l5io** [comment]
🚀 Preview for commit 7cc1c0ccea9e993264e74c5412e857da58272137 at: https://64d3908747ab9d6415d43832--layer5.netlify.app

File: src/sections/Community/Handbook/repo-data.js [hunk 1]
```diff
@@ -126,6 +127,15 @@ export const repo_data = [
         link: ["https://layer5.io/community/members/pranav-singh"],
         repository: "https://github.com/layer5io/meshery-cloud",
         accessRequired: "*program participation required"
+      },
+      {
+        project: "Recognition Program",
+        image: recognitionLogo,
+        site: "https://badges.layer5.io/",
+        language: "Gatsby",
+        maintainers_name: ["Vacant"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:136]: Let's hope this changes soon.

File: src/sections/Community/Handbook/repo-data.js [hunk 2]
```diff
@@ -126,6 +127,15 @@ export const repo_data = [
...
+        link: ["#"],
```
Comment (leecalcote) [inline src/sections/Community/Handbook/repo-data.js:137]:
```suggestion
        link: ["https://badges.layer5.io"],
```

**Review by leecalcote — APPROVED** [review]

File: src/sections/Community/Handbook/repo-data.js (hunk 1 above)
Comment (gopivaibhav) [reply src/sections/Community/Handbook/repo-data.js:136]: I am eagerly waiting for that day!! 🚀

**Pull request merged**